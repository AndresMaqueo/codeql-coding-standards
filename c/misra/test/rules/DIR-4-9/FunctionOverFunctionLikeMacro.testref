c/common/test/rules/functionlikemacrosdefined_shared/FunctionLikeMacrosDefined_shared.ql