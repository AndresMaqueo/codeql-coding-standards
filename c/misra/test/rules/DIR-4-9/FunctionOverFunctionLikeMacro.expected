| test.c:4:1:6:12 | #define MACRO4(L) (L" " "suffix") | Macro used when function call would be preferred. |
